crates/pkgcraft-tools/src/subcmds/pkg/source.rs

```diff
@@ -313,13 +312,13 @@ fn benchmark(bench: Bench, targets: PkgTargets, cmd: &Command) -> anyhow::Result
         };
 
     let pkgs = targets.ebuild_raw_pkgs();
-    let mut sorted = if cmd.sort { Some(vec![]) } else { None };
     let stdout = io::stdout().lock();
     let mut out = cmd.format.make_output(stdout);
 
-    for result in pkgs.par_map(func).jobs(cmd.jobs) {
-        match result {
-            Ok((pkg, data)) => {
+    let result: pkgcraft::Result<Vec<_>> = pkgs
```

Comment:
As this can be run across the entire repo, I'd rather not collect results into a vector.